Makefile

```diff
@@ -26,6 +26,10 @@ APPLICATION_NAME = $(shell jq -r '.displayName' package.json)
 APPLICATION_DESCRIPTION = $(shell jq -r '.description' package.json)
 APPLICATION_COPYRIGHT = $(shell cat electron-builder.yml | shyaml get-value copyright)
 
+BINTRAY_ORGANIZATION = resin-io
+BINTRAY_REPOSITORY_DEBIAN = debian
+BINTRAY_REPOSITORY_REDHAT = rpm
```

Comment:
Hmm, I see that on https://bintray.com/resin-io you've got `debian` and `rpm`. If this is just because "that's the way that bintray works" (which again seems inconsistent, but ho-hum) then I guess we could either refer to `redhat` in the `Makefile` and then convert it to `rpm` inside `bintray.sh` ; or another approach would be to rename `BINTRAY_REPOSITORY_REDHAT` to `BINTRAY_REPOSITORY_RPM`.